src/ecalc/libraries/libecalc/common/libecalc/input/yaml_types/components/legacy/yaml_fuel_consumer.py

```diff
@@ -1,37 +1,14 @@
-from typing import Any, Dict, Type
-
-from libecalc.dto.base import ConsumerUserDefinedCategoryType
-from libecalc.input.yaml_types import YamlBase
-from libecalc.input.yaml_types.components.category import CategoryField
-from libecalc.input.yaml_types.placeholder_type import PlaceholderType
+from libecalc.input.yaml_types.components.legacy.yaml_electricity_consumer import (
+    YamlElectricityConsumer,
+)
 from libecalc.input.yaml_types.temporal_model import TemporalModel
 from pydantic import Field
 
 
-class YamlFuelConsumer(YamlBase):
+class YamlFuelConsumer(YamlElectricityConsumer):
```

Comment:
```suggestion
class YamlFuelConsumer(YamlElectricityConsumer):
```
not sure what you are thinking here. Doesnt this add some fields from `YamlElectricityConsumer` that we do not want?

Reply:
No, the only difference I found was fuel. We can split if that isn't the case

Reply:
I think so, or alternatively add a new "top YamlConsumer" that is abstract...since this inheritance is very weird...

Reply:
a new top YamlLegacyConsumer could work I guess, then we'd get 
```
class YamlElectricityConsumer(YamlLegacyConsumer):
    pass
```

I copy-pasted, no common parent